zcash_client_backend/src/data_api.rs

```diff
@@ -302,6 +302,30 @@ pub struct SentTransactionOutput {
     pub value: Amount,
     /// The memo that was attached to the output, if any
     pub memo: Option<MemoBytes>,
+    /// The decrypted note, if the sent output belongs to this wallet
+    sapling_change_to: Option<(AccountId, sapling::Note)>,
```

Comment:
The fact that `SentTransactionOutput` is no longer directly constructable will need a mention in the changelog (with "use `SentTransactionOutput::from_parts` instead").